daemons/data_collector.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import asyncio
+
+from usecases.collect_nodes_statuses import CollectNodesStatuses
+
+
+class DataCollector:
+
+    async def run(self) -> None:
+        await CollectNodesStatuses().collect()
+
+
+def main() -> None:
```

Comment:
Is this needed? I think the idea is to run `DataCollector` from the `daemons_runner`, right?

Reply:
It's good for testing in dev and in case we need to run a single daemon for any reason (fix data or something). But i can remove if it's a bad idea.

Reply:
No need to remove, I am ok with it.